doc/reference/configuration/configuration_reference.rst

```diff
@@ -2988,6 +2988,73 @@ The ``memtx`` section is used to configure parameters related to the :ref:`memtx
     | Default: box.NULL
     | Environment variable: TT_MEMTX_SORT_THREADS
 
+
+
+
+..  _configuration_reference_metrics:
+
+metrics
+-------
+
+The ``metrics`` section defines configuration parameters for :ref:`metrics <monitoring>`.
+
+..  NOTE::
+
+    ``metrics`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`metrics.exclude <configuration_reference_metrics_exclude>`
+-   :ref:`metrics.include <configuration_reference_metrics_include>`
+-   :ref:`metrics.labels <configuration_reference_metrics_labels>`
+
+
+.. _configuration_reference_metrics_exclude:
+
+..  confval:: metrics.exclude
+
+    Metrics to turn off.
+    This option accepts the same values as the ``exclude`` configuration parameter passed to :ref:`metrics.cfg() <metrics_cfg>`.
+
+    **Example**
+
+    ..  literalinclude:: /code_snippets/snippets/sharding/instances.enabled/sharded_cluster_crud_metrics/config.yaml
+        :start-at: metrics:
+        :end-at: instance_name
+        :language: yaml
+        :dedent:
+
+    |
+    | Type: array
+    | Default: nil
+    | Environment variable: TT_METRICS_EXCLUDE
+
+
+.. _configuration_reference_metrics_include:
+
+..  confval:: metrics.include
+
+    Metrics to turn on.
+    This option accepts the same values as the ``include`` configuration parameter passed to :ref:`metrics.cfg() <metrics_cfg>`.
+
+    |
+    | Type: array
+    | Default: nil
+    | Environment variable: TT_METRICS_INCLUDE
+
+
+.. _configuration_reference_metrics_labels:
+
+..  confval:: metrics.labels
+
+    Global :ref:`labels <metrics-api_reference-labels>` to be added to every observation.
+
+    |
+    | Type: map
+    | Default: nil
```

Comment:
Actually it's `{ alias = names.instance_name }` map (and behavior with `nil` is actually different).

https://github.com/tarantool/tarantool/blob/0b014ad3693bf41ffd25a27186fad8a53cc37205/src/box/lua/config/applier/box_cfg.lua#L365

Reply:
Hmm, output for metrics env variables shows `nil` for `TT_METRICS_LABELS`.. This might confuse if the behavior with nil is different.

```
 % tarantool --help-env-list | grep TT_METRICS
| TT_METRICS_EXCLUDE                                    | array          | nil                                           | 
| TT_METRICS_INCLUDE                                    | array          | nil                                           | 
| TT_METRICS_LABELS                                     | map            | nil                                           | 
```

Reply:
I'm not sure what env default are, but if the instance is bootstrapped with config and labels are not provided neither through config nor through env, `{ alias = names.instance_name }` is used.

Reply:
OK, updated in docs:

<img width="479" alt="image" src="https://github.com/user-attachments/assets/265d1137-61c4-40f2-b375-2c80f927428e">